ilamb_data/reverse_gpp_dim.jl

```diff
@@ -0,0 +1,17 @@
+using NCDatasets
+
+"""
+    gpp_reverse_dim()
+
+Reverse the dimensions for .nc file located at
+https://www.ilamb.org/ILAMB-Data/DATA/gpp/FLUXCOM/gpp.nc because the latitude dimension is
+in decreasing order.
+"""
+function gpp_reverse_dim()
+    ds = NCDataset("gpp.nc", "a")
+    dslat = reverse(copy(Array(ds["lat"])))
+    dsgpp = reverse(copy(Array(ds["gpp"])), dims = 2)
```

Comment:
```suggestion
    dslat = reverse(Array(ds["lat"]))
    dsgpp = reverse(copyArray(ds["gpp"]), dims = 2)
```
Is the copy required?